docs/modules/examples/pages/colbert.adoc

```diff
@@ -15,48 +15,55 @@ Use ColBERT, Astra DB, and RAGStack to:
 
 For more information, see xref:colbert:index.adoc[].
 
-== Prerequisites
-
 [NOTE]
 ====
 To run ragstack-ai-colbert in a Windows environment, use https://learn.microsoft.com/en-us/windows/wsl/install[Windows Subsystem for Linux].
 ====
 
-Import the ragstack-ai-colbert package:
+== Prerequisites
+
+. Install dependencies:
++
 [source,python]
 ----
-pip install ragstack-ai-colbert
+pip install ragstack-ai-colbert python-dotenv
+----
++
+. Create a `.env` file in your application directory with the following environment variables:
++
+[source,bash]
+----
+ASTRA_DB_APPLICATION_TOKEN=AstraCS: ...
+ASTRA_DB_ID=2eab82dc-9032-45ba-aeb0-a481b6f9458d
```

Comment:
maybe don't use an actual DB ID but show an example one? your-db-id-here

Reply:
It's a fake string